guardian/thrift-serializer
# Address node.js vulnerabilities and deprecations

## What does this change?

- Fix up vulnerable dependencies in the node.js package
- Fix `new Buffer()` deprecation warning

## How to test

The functionality should be unchanged, but no warnings shown when you run `npm install`

## How can we measure success?
Less deprecation warnings

## Have we considered potential risks?

n/a
